kwave/utils/conversion.py

```diff
@@ -15,7 +15,7 @@
 
 
 @beartype
-def db2neper(alpha: NUMERIC, y: NUMERIC = 1) -> float:
```

Comment:
We should decide if we want to write our k-wave types as e.g. `kt.NUMERIC` or import them as global constants e.g. `NUMERIC`. Once we decide, we should make this consistent through the code-base.

Reply:
I am in favor of kt.NUMERIC

Reply:
Agree on `kt.NUMERIC` :+1: